gnutls/gnutls-3.6.3-msysize.patch

```diff
@@ -2,7 +2,7 @@ diff -Naur gnutls-3.6.3-orig/build-aux/ar-lib gnutls-3.6.3/build-aux/ar-lib
 --- gnutls-3.6.3-orig/build-aux/ar-lib	2018-07-13 08:14:46.000000000 +0300
 +++ gnutls-3.6.3/build-aux/ar-lib	2018-07-26 13:56:44.494565900 +0300
 @@ -56,6 +56,9 @@
- 	  CYGWIN*)
+ 	  CYGWIN* | MSYS*)
  	    file_conv=cygwin
  	    ;;
 +	  MSYS*)
```

Comment:
Seems like this hunk could be removed now

Reply:
Ya, I realized that after pushing that. But this may not cause severe issue. I shall remove it next update.